include/fs_constants.hpp

```diff
@@ -0,0 +1,23 @@
+#ifndef FS_CONSTANTS_HPP
+#define FS_CONSTANTS_HPP
+
+#include <fcntl.h>
+#include <inttypes.h>
+#include <linux/fs.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/ioctl.h>
+#include <unistd.h>
+
+
+#define IO_BLOCK_SIZE 4096
+
+#define NUM_INODE_BLOCKS 1023
+#define NUM_BLOCKS 2048
+
+#define INODE_SIZE 512
+
+#define DATABLOCKS_PER_BITMAP_BLOCK 255

```

Comment:
I think we have 2047 datablocks in every bitmap block and it costs 255 bytes to represent 2047 bits.
Not sure about this.

Reply:
Good catch, I will fix. Why do we only store 2047 datablocks - why not 32704?

Reply:
Good point, we can consider modifying this if we do find we need more performance and further exploration.